fallout/gatling/gatling_timeseries_latency.yaml

```diff
@@ -0,0 +1,84 @@
+# master uses a Gatling DSE plugin based on 3.x
+# DSP-17972-unified-driver uses a Gatling DSE plugin based on the unified driver (>= 4.5.x)
+branch: "master"
+
+---
+
+ensemble:
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.instance.type: m3.large
+    configuration_manager:
+      - name: ctool
+        properties:
+          product.type: cassandra
+          product.install.type: tarball
+          product.version: "3.11.6"
+  client:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.instance.type: m3.xlarge
+    configuration_manager:
+      - name: ctool
+        properties:
+          install.maven: true
+      - name: gatling
+        properties:
+          git.repository: git@github.com:riptano/dse-benchmarks.git
+          git.branch: {{branch}}
+
+workload:
+  phases:
+    # This simulation isn't necessary in a single-node case; Timeseries_Write will
+    # create the infrastructure that's needed.  We do need it in the multi-node
+    # client case; otherwise you can see weird timing errors around the existence
+    # of the expected keyspace/table on various nodes.
+    - timeseries-schema:
+        module: gatling
+        properties:
+          maven_args: >
+              scala:testCompile
+              gatling:execute
+              -Dgatling.disableCompiler
+              -Dgatling.simulationClass=core.timeseries.Timeseries_Schema
+    - timeseries-write:
+        module: gatling
+        properties:
+          maven_args: >
+              scala:testCompile
+              gatling:execute
+              -Dgatling.disableCompiler
+              -Dgatling.simulationClass=core.timeseries.Timeseries_Write
+              -DconnectionsPerHost=8
+              -DrowsPerPartition=1000
+              -DinjectionType=constant
+              -Dthroughput=10000
```

Comment:
This parameter should be a lot higher. Kafka or dsbulk are able to generate traffic that is > 200_000 per second. We should validate at least this number of requests

Reply:
This value (like the timeout mentioned above) will also need some tweaking once the mechanism is fully in place and working.